test/unit/video.test.js

```diff
@@ -59,6 +59,15 @@ test('should add the value to the languages object with lower case lang code', f
   deepEqual(result, globalOptions['languages'][code.toLowerCase()], 'should also match');
 });
 
+test('should allow setting global options', function() {
+  videojs.setGlobalOptions({
```

Comment:
I agree it feels a little weird, but if it did an overwrite you'd have to include every option every time instead of being able to just set one or a few.